pythondata_cpu_vexriscv/verilog/src/main/scala/vexriscv/GenCoreDefault.scala

```diff
@@ -104,7 +106,7 @@ object GenCoreDefault{
             config = InstructionCacheConfig(
               cacheSize = argConfig.iCacheSize,
               bytePerLine = 32,
-              wayCount = 1,
+              wayCount = (argConfig.iCacheSize + 4095) / 4096,
```

Comment:
This isn't required when the MMU isn't used (linux == false)
So would be better to not enforce the use of multiple way if not required (for both D$ and I$)

Reply:
Thanks @Dolu1990 for taking the time to look.   I'm trying to figure this out, but if I leave wayCount=1, even the "Full" config (non-linux) gives me an error when I try to build with cache sizes of 8192; somehow the assertion at line 56 of `IBusCachedPlugin.scala` thinks I'm trying to build with an MMU.

Reply:
If I compare `memoryTranslatorPortConfig` to null, the result is that it is not null, which doesn't make sense to me, since `linux` **is** false.

Reply:
This particular issue seems to get resolved if I add "else null" as below:
```
memoryTranslatorPortConfig = if(linux) MmuPortConfig(portTlbSize = 4) else null
```
I'll push a commit that includes this along with a change to the wayCount calculation.

Reply:
I've made the change now, ready for review.

Reply:
Hoo right, that's something not so great in VexRiscv, i should fix that too.
Thanks for the workaround